linux-ppc64le-anvil/Dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM ppc64le/centos:7
+
+MAINTAINER conda-forge <conda-forge@googlegroups.com>
+
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Add a timestamp for the build. Also, bust the cache.
+ADD http://tycho.usno.navy.mil/timer.html /opt/docker/etc/timestamp
+
+# Resolves a nasty NOKEY warning that appears when using yum.
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+# Install basic requirements.
+RUN yum update -y && \
+    yum install -y \
+                   file \
+                   bzip2 \
+                   make \
+                   patch \
+                   sudo \
+                   tar \
+                   which \
+                   libXau \
+                   libxcb \
+                   libXdmcp \
+                   libXext-devel \
+                   libXrender-devel \
+                   libSM-devel \
+                   libX11-devel \
+                   libXt \
+                   mesa-libGL-devel \
+                   mesa-libGLU \
+                   openssl \
+                   openssh-clients \
+                   rsync \
+                   util-linux \
+                   wget \
+                   xorg-x11-server-Xvfb && \
+    yum clean all
+
+# give sudo permission for conda user to run yum (user creation is postponed
+# to the entrypoint, so we can create a user with the same id as the host)
+RUN echo 'conda ALL=NOPASSWD: /usr/bin/yum' >> /etc/sudoers
+
+# Install the latest Miniconda with Python 3 and update everything.
+RUN curl -s -L https://repo.continuum.io/miniconda/Miniconda3-4.3.27-Linux-ppc64le.sh > miniconda.sh && \
+    openssl md5 miniconda.sh | grep c51bc5fb00da12b487959b5adf0133ab && \
+    bash miniconda.sh -b -p /opt/conda && \
+    rm miniconda.sh && \
+    touch /opt/conda/conda-meta/pinned && \
+    export PATH=/opt/conda/bin:$PATH && \
+    conda config --set show_channel_urls True && \
+    conda config --add channels conda-forge && \
+    conda config --add channels furmanek && \
```

Comment:
I've copied su-exec to conda-forge for now. Can you remove this?

Reply:
FWIW we also package `su-exec` and could presumably pick up PPC builds for it after we get this sorted.

Had debated switching our Docker image to `su-exec`, but was unclear on it's maintenance status. As `gosu` had a better maintenance record, stuck with it. We can revisit though.

ref: https://github.com/conda-forge/su-exec-feedstock

Reply:
In any event, `su-exec` is a good starting point as it is C only and `gosu` requires a working Go compiler.